lib/runner.js

```diff
@@ -123,16 +123,22 @@ module.exports = Runner;
  * @class
  * @param {Suite} suite Root suite
  * @param {boolean} [delay] Whether to delay execution of root suite until ready.
+ * @param {boolean} [cleanReferencesAfterRun] Whether to clean references to test fns and hooks when a suite is done.
  */
-function Runner(suite, delay) {
+function Runner(suite, delay, cleanReferencesAfterRun) {
   var self = this;
   this._globals = [];
   this._abort = false;
   this._delay = delay;
   this.suite = suite;
   this.started = false;
+  this._cleanReferencesAfterRun = cleanReferencesAfterRun;
   this.total = suite.total();
   this.failures = 0;
+  /**
+   * @types {[[EventEmitter, string, function]]}
+   */
```

Comment:
`@types` isn't understood by JSDoc.  afaik there's no way to define a tuple either. the best you can get is `@type {(EventEmitter|string|function)[]}`, I think.